source/intro.tex

```diff
@@ -950,16 +950,16 @@
 \pnum
 \defn{Sequenced before} is an asymmetric, transitive, pair-wise relation between
 evaluations executed by a single thread~(\ref{intro.multithread}), which induces
-a partial order among those evaluations. Given any two evaluations \term{A} and
-\term{B}, if \term{A} is sequenced before \term{B}
-(or, equivalently, \term{B} is \defn{sequenced after} \term{A}),
+a partial order among those evaluations. Given any two evaluations \placeholder{A} and
+\placeholder{B}, if \placeholder{A} is sequenced before \placeholder{B}
```

Comment:
This is OK for now. See Issue #1060, I may eventually wish to move away from using `\placeholder` both in code and in textual situations, but I'm not sure yet how.